en/docs/administer/product-administration/monitoring/jmx-based-monitoring.md

```diff
@@ -8,24 +8,24 @@ JMX is enabled in WSO2 products by default, which ensures that the JMX server st
 
 ### Configuring JMX ports for the server
 
-The default JMX ports (RMIRegistryPort and the RMIServerPort) are configured in the `carbon.xml` file (stored in the `<PRODUCT_HOME>/repository/conf` directory) as shown below. If required, you can update these default values.
+The default JMX ports (RMIRegistryPort and the RMIServerPort) can be changed by adding the below config into 
+`<PRODUCT_HOME>/repository/conf/deployment.toml`file as shown below. Below are the default values and you can update 
```

Comment:
`<PRODUCT_HOME>/repository/conf/deployment.toml`file as shown below.

There should be a space before **'file'**

Reply:
Below are the default values and you can update these default values as required. => Following are the default values and you can update them as required

Reply:
Fixed with 5086532aee5e568eda5989913fb757c3f564c6f3